### ISSUE TITLE
I want to build a corresponding movfuscator on the ARM architecture. How should I pay attention to it or how to carry out this project

### ISSUE THREAD

**Issue by dnffabs** [xoreaxeaxeax/movfuscator#48]
I want to apply this project to the ARM architecture. What should I pay attention to, how should I plan the entire process, and can the author provide some feedback

**Comment by edcdecl** [comment]
you can't. the entire reason this is possible on x86 is because the mov instruction can do math in its operands.

**Comment by gogo2464** [comment]
I do not think it could be `mathematically` impossible. We can alway use arm code and temp stack variable. I still think it will be hard to implement.

I would like to vcreate a study group for : https://www.reddit.com/r/REMath/ do you wanna join us? :)

**Comment by edcdecl** [comment]
> I do not think it could be `mathematically` impossible. We can alway use arm code and temp stack variable. I still think it will be hard to implement.
> 
> 
> 
> I would like to vcreate a study group for : https://www.reddit.com/r/REMath/ do you wanna join us? :)

it's not impossible but it wouldn't be the same. the whole reason this works is because it abuses x86 by doing everything through lookup tables. you could definitely make a compiler that is only allowed to generate a couple of instructions that make lookup tables possible, and the resulting binary would most likely still look really obfuscated.

**Comment by aidv** [comment]
I'd like to have a movfuscator for ARM

**Comment by gogo2464** [comment]
if I could I would
